src/mon/OSDMonitor.cc

```diff
@@ -1073,14 +1130,17 @@ void OSDMonitor::prime_pg_temp(
   const OSDMap& next,
   pg_t pgid)
 {
-  if (mon->monmap->get_required_features().contains_all(
+  if (mon->monmap->get_required_features().empty()) {
+    dout(10) << __func__ << ": quorum not formed" << dendl;
```

Comment:
@tchaikov, don't we have a way to directly query the quorum status, instead of this indirect feature query?